elastalert/alerts.py

```diff
@@ -769,10 +780,26 @@ def alert(self, matches):
                         self.comment_on_ticket(ticket, match)
                     except JIRAError as e:
                         logging.exception("Error while commenting on ticket %s: %s" % (ticket, e))
+                    if self.labels:
+                        for l in self.labels:
+                            try:
+                                self.issue = self.client.issue(ticket)
```

Comment:
Doesn't this return the same object as ticket? IE, could just do ``ticket.fields.labels.append(l)``?